### PR TITLE
Add a description about adding a new attribute on the fly while creating a product

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adda a description about adding a new attribute on the fly while creating a product

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/product-create-grouped.html

### CODE REVIEW

File: src/catalog/product-create-grouped.md [hunk 1]
```diff
@@ -34,6 +34,21 @@ The form is updated to reflect the change.
...
+   ![]({% link images/images/product-attribute-add-select.png %}){: .zoom}
+   _New Attribute_
+
+1. To add an existing attribute to the product, use the [filter controls]({% link stores/admin-grid-controls.md %}) to find the attribute in the grid and do the following:
+
+   - Select the checkbox in the first column of each attribute to be added.
+
+   - Click <span class="btn">Add Selected</span>.
+
+1. To define a new attribute, see [Adding an Attribute to a Product]]({% link catalog/product-attributes-add.md %})
```
Comment (leslietilling) [inline src/catalog/product-create-grouped.md:50]:
```suggestion
If the needed attributes do not exist, you can add new attributes while creating a product:

 - In the upper-right corner, click <span class="btn">Add Attribute</span>.
 - Define a new attribute (see [Adding an Attribute to a Product]]({% link catalog/product-attributes-add.md %})).

   ![]({% link images/images/product-attribute-add-select.png %}){: .zoom}
   _New Attribute_

To add an existing attribute to the product, use the [filter controls]({% link stores/admin-grid-controls.md %}) to find the attribute in the grid and do the following:

   - Select the checkbox in the first column of each attribute to be added.
   - Click <span class="btn">Add Selected</span>.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Edits and clarify create new vs. add from list

File: src/catalog/product-create-grouped.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-create-grouped.md]: Hi @leslietilling , thank you for reviewing.
I have implemented your suggestion, please approve this PR💙

Thank you!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**